config/esp32/components/chip/Kconfig

```diff
@@ -319,7 +319,7 @@ menu "CHIP Device Layer"
         config WIFI_STATION_RECONNECT_INTERVAL
             int "WiFi Station Interface Reconnect Interval (ms)"
             range 0 65535
-            default 5000
+            default 100
```

Comment:
100ms seems pretty aggressive for a reconnect interval, do we believe that wifi will reconnect in that short period of time if it had previously failed?

I think the default wifi reconnection interval for a lot of products is ~30s or so

If we believe decreasing this timeout will help, can we at least try to do something less aggressive and go down to 2s or 1s?

Reply:
As per our Wi-Fi team, the STA reconnection interval is not at all required and device can immediately retry. We have kept it as 100ms to be on the safer side. The original value of 5 sec was added during the first port of ESP32 and was never revisited since

Reply:
100ms does seem too fast, but it depends what it means under the hood. Definitely, attempting to reconnect within 100ms of having been disassociated is OK, but when associating, failure to associate should likely not cause infinite retry at 100ms interval.

Reply:
Given this is an Espressif engineer changing espressif code, based on their own analysis, approving.